java/libraries/io/src/processing/io/GPIO.java

```diff
@@ -165,7 +165,7 @@ protected static void checkValidPin(int pin) {
    *  the sketch is going to achieve.
    *  @param ms milliseconds to pause
    */ 
-  protected static void delay(int ms) {
+  public static void delay(int ms) {
```

Comment:
Thanks for pointing out - must have totally forgotten about the other one. Will remove. And since microseconds-type granularity is a little bit far fetched anyway, I guess the other one can go as well. Or perhaps a variant of delay() in PApplet that takes a float?